README.md

```diff
@@ -33,6 +33,10 @@ The SDK complies with three latest engine versions.
 
 - Currently automatic crash capturing works only for mobile platforms. In order to capture crahes on Windows and Mac
 [Crash Reporter has to be configured](https://docs.sentry.io/platforms/unreal/setup-crashreport/)
+  
+- Using UGS binaries requires tagging of files to ensure the crashpad_handler.exe is present. Such as the following. 
+  					<Tag Files="#EditorBinaries$(EditorPlatform)" Filter="*.target" With="#TargetReceipts"/>
+					<TagReceipt Files="#TargetReceipts" RuntimeDependencies="true" With="#BinariesToArchive$(EditorPlatform)"/>
```

Comment:
Do you know if there's any way this could be done automatically (from user perspective)?

Reply:
@vaind This would include it within the UGS Binaries zip. It would require adjustments to the build graph on each project (Or scripts)

We have just encountered an issue with this change locally though as it tries to replace the uproject file (Which is read only).

Reply:
@vaind Updated the wording and example.